src/app/productConfig/productConfigForm/productConfigForm.component.js

```diff
@@ -232,20 +253,30 @@ class ProductConfigFormController {
     }
   }
 
-  changeAmount (amount) {
+  changeAmount (amount, retainCoverFees) {
     this.itemConfigForm.$setDirty()
     this.checkAmountChanged(amount)
     this.itemConfig.amount = amount
     this.customAmount = ''
     this.customInputActive = false
+    if (!retainCoverFees && this.amountChanged) {
+      this.orderService.clearBrandedCoverFees()
+      this.itemConfig.coverFees = false
+      this.$scope.$emit(brandedCheckoutAmountUpdatedEvent)
+    }
     this.updateQueryParam({ key: giveGiftParams.amount, value: amount })
   }
 
-  changeCustomAmount (amount) {
+  changeCustomAmount (amount, retainCoverFees) {
     this.checkAmountChanged(amount)
     this.itemConfig.amount = amount
     this.customAmount = amount
     this.customInputActive = true
+    if (!retainCoverFees && this.amountChanged) {
```

Comment:
For some reason, this doesn't work the first time. Maybe `this.amountChanged` is false.

Reply:
It looks like `itemConfig.amount` is undefined.

Reply:
@OzzieOrca do you think I should default the amount to `50` (e.g at line 88) or change the logic for checking the amount changed to not require a defined itemConfig amount (line 285)? I'm leaning towards the latter.